src/main/java/MegaMillions.java

```diff
@@ -0,0 +1,83 @@
+import java.io.*;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MegaMillions {
+
+    File file = new File("src/main/resources/mega_millions.csv");
+    String absolutePath = file.getAbsolutePath();
+
+    public void filterAndPrintNumbers() {
+        List<Integer> allWinningNumbers = new ArrayList<>();
+        List<Integer> allWinningMegaBall = new ArrayList<>();
+        getAllWinningNumbers(allWinningNumbers, allWinningMegaBall);
+        List<Number> winning75Numbers = getUniqueNumbersWithOccurs(allWinningNumbers);
```

Comment:
Either use allWinning lists or winnging lists. We don't need both of them. If methods are returning new list with results why do you need to pass them empty arrays? It doesn't make sense.

Reply:
Oh, now I get it. They are actually filled because we invoke getAllWinningNumber.